TST: Test failures due to RGF93 v1 -> ETRS89-FRA [RGF93 v1]

https://github.com/pyproj4/pyproj/actions/runs/20102673098/job/57677657922
```
FAILED test/crs/test_crs.py::test_coordinate_operation__from_user_input[RGF93 v1 to WGS 84 (1)] - pyproj.exceptions.CRSError: Invalid coordinate operation string: RGF93 v1 to WGS 84 (1): (Internal Proj Error: proj_create: unrecognized format / unknown name)
FAILED test/crs/test_crs.py::test_coordinate_operation__from_string[urn:ogc:def:coordinateOperation:EPSG::1671] - AssertionError: assert 'ETRS89-FRA [...to WGS 84 (1)' == 'RGF93 v1 to WGS 84 (1)'
  
  - RGF93 v1 to WGS 84 (1)
  + ETRS89-FRA [RGF93 v1] to WGS 84 (1)
  ? ++++++++++++        +
FAILED test/crs/test_crs.py::test_coordinate_operation__from_string[RGF93 v1 to WGS 84 (1)] - pyproj.exceptions.CRSError: Invalid coordinate operation string: RGF93 v1 to WGS 84 (1): (Internal Proj Error: proj_create: unrecognized format / unknown name)
FAILED test/test_transformer.py::test_transformer_from_pipeline__input_types[EPSG:1671] - AssertionError: assert 'ETRS89-FRA [...to WGS 84 (1)' == 'RGF93 v1 to WGS 84 (1)'
  
  - RGF93 v1 to WGS 84 (1)
  + ETRS89-FRA [RGF93 v1] to WGS 84 (1)
  ? ++++++++++++        +
FAILED test/test_transformer.py::test_transformer_from_pipeline__input_types[RGF93 v1 to WGS 84 (1)] - pyproj.exceptions.ProjError: Invalid projection RGF93 v1 to WGS 84 (1).: (Internal Proj Error: proj_create: unrecognized format / unknown name)
FAILED test/test_transformer.py::test_transformer_from_pipeline__input_types[urn:ogc:def:coordinateOperation:EPSG::1671] - AssertionError: assert 'ETRS89-FRA [...to WGS 84 (1)' == 'RGF93 v1 to WGS 84 (1)'
  
  - RGF93 v1 to WGS 84 (1)
  + ETRS89-FRA [RGF93 v1] to WGS 84 (1)
  ? ++++++++++++        +
FAILED test/test_transformer.py::test_transformer_from_pipeline__wkt_json[to_wkt] - AssertionError: assert 'ETRS89-FRA [...to WGS 84 (1)' == 'RGF93 v1 to WGS 84 (1)'
  
  - RGF93 v1 to WGS 84 (1)
  + ETRS89-FRA [RGF93 v1] to WGS 84 (1)
  ? ++++++++++++        +
FAILED test/test_transformer.py::test_transformer_from_pipeline__wkt_json[to_json] - AssertionError: assert 'ETRS89-FRA [...to WGS 84 (1)' == 'RGF93 v1 to WGS 84 (1)'
  
  - RGF93 v1 to WGS 84 (1)
  + ETRS89-FRA [RGF93 v1] to WGS 84 (1)
  ? ++++++++++++        +
```

--- a/test/conftest.py
+++ b/test/conftest.py
@@ -15,6 +15,7 @@ PROJ_LOOSE_VERSION = version.parse(pyproj.__proj_version__)
 PROJ_GTE_941 = PROJ_LOOSE_VERSION >= version.parse("9.4.1")
 PROJ_GTE_95 = PROJ_LOOSE_VERSION >= version.parse("9.5.0")
 PROJ_GTE_971 = PROJ_LOOSE_VERSION >= version.parse("9.7.1")
+PROJ_GTE_980 = PROJ_LOOSE_VERSION >= version.parse("9.8.0")
 
 
 def unset_data_dir():

--- a/test/crs/test_crs.py
+++ b/test/crs/test_crs.py
@@ -23,6 +23,7 @@ from pyproj.transformer import TransformerGroup
 from test.conftest import (
     PROJ_GTE_941,
     PROJ_GTE_971,
+    PROJ_GTE_980,
     assert_can_pickle,
     grids_available,
 )
@@ -1222,7 +1223,7 @@ def test_coordinate_operation__from_string():
     co = CoordinateOperation.from_string("urn:ogc:def:coordinateOperation:EPSG::1671")
     # PROJ 9.7.1+ renamed this operation from "RGF93 v1 to WGS 84 (1)"
     # to "ETRS89-FRA [RGF93 v1] to WGS 84 (1)"
-    if PROJ_GTE_971:
+    if PROJ_GTE_980:
         assert co.name == "ETRS89-FRA [RGF93 v1] to WGS 84 (1)"
     else:
         assert co.name == "RGF93 v1 to WGS 84 (1)"

--- a/test/test_transformer.py
+++ b/test/test_transformer.py
@@ -19,7 +19,7 @@ from pyproj.enums import CRSExtentUse, IntermediateCRSUse, TransformDirection
 from pyproj.exceptions import ProjError
 from pyproj.transformer import AreaOfInterest, TransformerGroup
 from test.conftest import (
-    PROJ_GTE_971,
+    PROJ_GTE_980,
     grids_available,
     proj_env,
     proj_network_env,
@@ -1145,7 +1145,7 @@ def test_transformer_authority_filter():
 def test_transformer_from_pipeline__input_types(input_string):
     # PROJ 9.7.1+ renamed this operation from "RGF93 v1 to WGS 84 (1)"
     # to "ETRS89-FRA [RGF93 v1] to WGS 84 (1)"
-    if PROJ_GTE_971:
+    if PROJ_GTE_980:
         assert (
             Transformer.from_pipeline(input_string).description
             == "ETRS89-FRA [RGF93 v1] to WGS 84 (1)"
@@ -1173,7 +1173,7 @@ def test_transformer_from_pipeline__wkt_json(method_name):
             method_name,
         )()
     ).description
-    if PROJ_GTE_971:
+    if PROJ_GTE_980:
         assert description == "ETRS89-FRA [RGF93 v1] to WGS 84 (1)"
     else:
         assert description == "RGF93 v1 to WGS 84 (1)"

